x-pack/test_serverless/api_integration/test_suites/common/reporting/management.ts

```diff
@@ -8,62 +8,70 @@
-    this.tags(['failsOnMKI']);
-
-    const dataArchive = 'x-pack/test/functional/es_archives/reporting/archived_reports';
-
     before(async () => {
-      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
-    });
-
-    beforeEach(async () => {
-      await esArchiver.load(dataArchive);
+      adminUser = await svlUserManager.createApiKeyForRole('admin');
+      internalReqHeader = svlCommonApi.getInternalRequestHeader();
     });
-
     after(async () => {
-      await esArchiver.unload(dataArchive);
-      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
+      await svlUserManager.invalidateApiKeyForRole(adminUser);
     });
 
     describe('Deletion', () => {
-      const DELETE_REPORT_ID = 'krazcyw4156m0763b503j7f9';
+      let reportJob: ReportApiJSON;
 
-      // archived data uses the test user but functionality for specific users is not possible yet for svl
-      xit(`user can delete a report they've created`, async () => {
-        const response = await supertestWithoutAuth
-          .delete(`${INTERNAL_ROUTES.JOBS.DELETE_PREFIX}/${DELETE_REPORT_ID}`)
-          .set(...API_HEADER)
-          .set(...INTERNAL_HEADER)
-          .set(roleAuthc.apiKeyHeader);
+      const createJob = async (roleAuthc: RoleCredentials): Promise<ReportApiJSON> => {
+        log.info(`request report job with ApiKey ${adminUser.apiKey.name}`);
+        const { job } = await reportingAPI.createReportJobInternal(
+          'csv_searchsource',
+          {
+            browserTimezone: 'UTC',
+            objectType: 'search',
+            searchSource: {
+              index: '5193f870-d861-11e9-a311-0fa548c5f953',
+              query: { language: 'kuery', query: '' },
+              version: true,
+            },
+            title: 'Ecommerce Data',
+            version: '8.15.0',
+          },
+          roleAuthc,
+          internalReqHeader
+        );
+        log.info(`created report job ${job.id} with ApiKey ${adminUser.apiKey.name}`);
+        return job;
+      };
 
-        expect(response.status).to.be(200);
-        expect(response.body).to.eql({ deleted: true });
+      before(async () => {
+        reportJob = await createJob(adminUser);
       });
 
-      it(`user can not delete a report they haven't created`, async () => {
+      it(`user can delete a report they've created`, async () => {
+        // for this test, we don't need to wait for the job to finish or verify the result
         const response = await supertestWithoutAuth
-          .delete(`${INTERNAL_ROUTES.JOBS.DELETE_PREFIX}/${DELETE_REPORT_ID}`)
+          .delete(`${INTERNAL_ROUTES.JOBS.DELETE_PREFIX}/${reportJob.id}`)
           .set(...API_HEADER)
           .set(...INTERNAL_HEADER)
-          .set(roleAuthc.apiKeyHeader);
+          .set(adminUser.apiKeyHeader);
 
-        expect(response.status).to.be(404);
```

Comment:
This 404 response was due to the fact that the archive of reports was not correctly installed. Therefore, this test was not adequately testing the functionality.